security/tests/integration/secretCreationTest/secret_creation_test.go

```diff
@@ -82,4 +83,4 @@ func TestMain(m *testing.M) {
 	glog.Infof("Test result %d in env %s", res, testEnvName)
 
 	os.Exit(res)
-}
+}
```

Comment:
missing newline (set your editor preferences to always end last line with a new line)

Reply:
Good tip! Set it.